plugin/data_masking/src/plugin.cc

```diff
@@ -67,3 +74,17 @@ mysql_declare_plugin(data_masking){
     NULL,                            // config options
     0,                               // flags
 } mysql_declare_plugin_end;
+
+// Returns true and populates the msg with an error if the module has not been initialized
+bool data_masking_validate_status(char *msg, size_t msg_len) {
+  if (!data_masking_init) {
+    strncpy(
+        msg,
+        "This function requires data_masking plugin which is not installed."
+        " Please install keyring_udf plugin and try again.",
+        msg_len - 1);
+    msg[msg_len - 1]= '\0';
```

Comment:
Maybe use std::snprintf? (I don't see any advantage but just for consistency reason)